.github/workflows/release.yml

```diff
@@ -7,6 +7,7 @@ on:
 
 jobs:
   goreleaser:
+    permissions: write-all
```

Comment:
In theory this is not needed?

It already worked without this on both this repo and others.

Reply:
Perhaps it's a different setting between these repositories. I'd prefer that you delete the last commit and, if on release I get the same error, then I can add it back.

Reply:
Sure. I've reverted last commit